addons/website_forum/controllers/main.py

```diff
@@ -125,6 +126,11 @@ def questions(self, forum, tag=None, page=1, filters='all', my=None, sorting=Non
                 Post._generate_order_by(sorting, None)
```

Comment:
We should probably just url decode the sorting param?
And worst case just add the UserError next to the ValueError except so we sort a page without sorting, better than a 404 :+1:

Reply:
@rdeodoo Thanks for your suggestion, I've fixed it. Could u take a look?